data/tabular/rdkit_features/meta.yaml

```diff
@@ -0,0 +1,151 @@
+      Assistant: {#Cool, |Nice, |Interesting, |That's interesting, |That is a very interesting question, |!}Do you have some additional {#contraints|requirements|conditions|limitations|!}{# I should take into account| that help me narrow down the search| that I should consider| I should consider| I should take into account|!}?
+      User: {#Yes, |Indeed, |Yeah, |Yea, |Yep, |!}I want the {NumHDonors__names__noun} to be {NumHDonors#}, the {NumHAcceptors__names__noun} to be {NumHAcceptors#}.
+      Assistant: {#Then, |In that case, |In that situation, |In that scenario, |!}I {#recommend|suggest|propose|advise|!} the {#molecule|compound|chemical!} with {SMILES__description} {SMILES#}.
+    - |-
+      Question: What is the {formula__names__noun} of the {#molecule|compound|chemical!} with {SMILES__description} {SMILES#}?
+      Answer: {formula#}.
+    - |-
+      Question: What is the {NumHDonors__names__noun} of the {#molecule|compound|chemical!} with {SMILES__description} {SMILES#}?
+      Answer: {NumHDonors#}.
+    - |-
+      Question: What is the {NumHAcceptors__names__noun} of the {#molecule|compound|chemical!} with {SMILES__description} {SMILES#}?
+      Answer: {NumHAcceptors#}.
+    - |-
+      Question: What is the {NumHeteroatoms__names__noun} of the {#molecule|compound|chemical!} with {SMILES__description} {SMILES#}?
+      Answer: {NumHeteroatoms#}.
+    - |-
+      Question: What is the {RingCount__names__noun} of the {#molecule|compound|chemical!} with {SMILES__description} {SMILES#}?
+      Answer: {RingCount#}.
+    - |-
+      Question: What is the {NumRotatableBonds__names__noun} of the {#molecule|compound|chemical!} with {SMILES__description} {SMILES#}?
+      Answer: {NumRotatableBonds#}.
+    - |-
+      Question: What is the {NumAromaticBonds__names__noun} of the {#molecule|compound|chemical!} with {SMILES__description} {SMILES#}?
+      Answer: {NumAromaticBonds#}.
+    - |-
+      Question: What is the {NumAcidGroups__names__noun} of the {#molecule|compound|chemical!} with {SMILES__description} {SMILES#}?
+      Answer: {NumAcidGroups#}.
+    - |-
+      Question: What is the {NumBasicGroups__names__noun} of the {#molecule|compound|chemical!} with {SMILES__description} {SMILES#}?
+      Answer: {NumBasicGroups#}.
+    - |-
+      Question: What is the {Apol__names__noun} of the {#molecule|compound|chemical!} with {SMILES__description} {SMILES#}?
+      Answer: {Apol#}.
+    - |-
+      User: I want to {#design|create|make|synthesize|analyze!} a {#molecule|compound|chemical!} with a {NumHDonors__names__noun} of {NumHDonors#} and a {NumHAcceptors__names__noun} of {NumHAcceptors#}.
+      Assistant: {#Cool, |Nice, |Interesting, |That's interesting, |That is a very interesting question, |!}Do you have some additional {#contraints|requirements|conditions|limitations|!}{# I should take into account| that help me narrow down the search| that I should consider| I should consider| I should take into account|!}?
```

Comment:
```suggestion
      Assistant: {#Cool, do|Nice, do|Interesting, do|That's interesting, do|That is a very interesting question, do|Do!} you have some additional {#contraints|requirements|conditions|limitations|!}{# I should take into account| that help me narrow down the search| that I should consider| I should consider| I should take into account|!}?
```

Reply:
This needs to be fixed in other templates too.

Reply:
wow, that is a great catch. I didn't even spot it when reading the generated text